launcher/src/store/services.js

```diff
@@ -423,9 +423,17 @@ export const useServices = defineStore("services", {
           hIcon: "/img/icon/service-icons/prometheus.png",
           linkUrl: "",
           docsUrl: "https://prometheus.io/docs/introduction/overview/",
-          headerOption: true,
+          headerOption: false,
```

Comment:
Do you need to change this?

Reply:
with this open are you removing the header icon:
<img width="152" alt="grafik" src="https://user-images.githubusercontent.com/80448387/230047509-bd8a9c65-04a5-476a-bcc0-2a75f67139ee.png">

Reply:
Accidentally left this change after testing something